PROJECT_CHARTER_TEMPLATE.md

```diff
@@ -21,7 +21,7 @@ _to a colleague who is familiar with related technical concepts but unfamiliar_
 _with the project. You may also want to describe the project's value to community_
 _and/or business stakeholders._
 
-ex. [K8s SIG Architecture Charter](https://github.com/kubernetes/community/blob//blob/HEAD/sig-architecture/charter.md#scope)
+ex. [K8s SIG Architecture Charter](https://github.com/kubernetes/community/blob/master/sig-architecture/charter.md#scope)
```

Comment:
these are intentionally kept as "HEAD", despite the redirect, to ensure they work regardless of the default branch name.

Reply:
fixed :)